msg_helpers.go

```diff
@@ -783,11 +783,7 @@ func unpackDataAplPrefix(msg []byte, off int) (APLPrefix, int, error) {
 	if int(prefix) > 8*len(ip) {
 		return APLPrefix{}, len(msg), &Error{err: "APL prefix too long"}
 	}
-
 	afdlen := int(nlen & 0x7f)
-	if (int(prefix)+7)/8 != afdlen {
```

Comment:
Removing this check and adding a new one below will allow `last := ip[afdlen-1]` below to panic.

Reply:
thanks, update last octet check.